driver/registry_base.go

```diff
@@ -431,6 +432,24 @@ func (m *RegistryBase) SubjectIdentifierAlgorithm() map[string]consent.SubjectId
 	return m.sia
 }
 
+func (s *RegistryBase) ObfuscateSubjectIdentifier(cl fosite.Client, subject, forcedIdentifier string) (string, error) {
```

Comment:
I would suggest to move this to `ConsentStrategy` because the registry is responsible for instantiation, not for business logic. That way, you can also just pass the consent request which has both `subject` as well as `forcedIdentifier`

Reply:
Alright, I moved in commit bb34087. I haven't changed the interface to take a consent request because the function is called from a place where we'd then need to create a request stub just to call the function